ioexpander/__init__.py

```diff
@@ -167,11 +167,12 @@ def __init__(
         interrupt_pin=None,
         interrupt_pull_up=False,
         gpio=None,
+        smbusnr=1,
         skip_chip_id_check=False,
         perform_reset=False
     ):
         self._i2c_addr = i2c_addr
-        self._i2c_dev = SMBus(1)
+        self._i2c_dev = SMBus(smbusnr)
```

Comment:
Thank you for this. Any chance you can change the variable name to something like `smbus_id`. Something more descriptive. Thanks